.github/pull_request_template.md

```diff
@@ -1,23 +1,16 @@
 ## Summary
-Provide details about your pull request and what it adds, fixes, or changes. Photos and videos are recommended.
+> Describe what the PR does. And how to test. Photos and videos are recommended.
```

Comment:
Should we also mention that you should remove sections that are not relevant?

Reply:
all the sections are relevant now.. tickets should almost always be there. description is always needed. checklist is okay if it's there but not fully completed.. 
I am okay with default text making it into normal PRs.. I don't expect engineers to clean up the description every time.